scipy/constants/tests/test_codata.py

```diff
@@ -1,17 +1,15 @@
-from scipy.constants import find, value, ConstantWarning, c, speed_of_light
```

Comment:
Does this mean that `c, speed_of_light` are not available anymore? Because we cannot remove things from the API without a deprecation cycle.

Reply:
In my understanding they should still be available in `scipy.constants` as I only removed their import in the test file.

Reply:
oh ok I will test locally. I was not sure because you removed the line `c = value('speed of light in vacuum')`. I will pull your changes and play around.